### ISSUE TITLE
Trying to delete an active (frozen) job throws a 500

### ISSUE THREAD

**Issue by damusix** [taskforcesh/issues#62]
![Screenshot from 2021-04-07 18-49-37](https://user-images.githubusercontent.com/4218587/113944137-0ce20f00-97d2-11eb-8e23-d80b6e41e860.png)

**Comment by manast** [comment]
Thanks for reporting this.

**Comment by damusix** [comment]
@manast I had like 20 jobs that were stuck in "active" but weren't getting picked back up by any workers. Is there a way to resume jobs that are in an eternal active state? I suspect it happened due to blue/green deployments not gracefully shutting down bullmq or something. It seems to happen every so often too.

In any case, I found this issue because I had to reschedule a new job and delete the old one.

**Comment by manast** [comment]
Normally if they are active and the worker processing the job dies, the jobs will automatically be moved to wait again and picked by other workers. Are you using Bull or BullMQ? In the later for this mechanism to work you need to have at least one instance of the "QueueScheduler" (https://docs.bullmq.io/guide/jobs/stalled). On the former this mechanism does not require any extra instaces. However the default max stalled retry attempt is 1, so if this happens more than once for the same job the job will fail, however in your case they have not left the active state so I assume you are using bullMQ but forgot to instantiate a QueueScheduler...

**Comment by damusix** [comment]
Yes... it is mentioned in docs. I'll get back to you when we I implement these if the issue persists.
